Nos/Views/NoteButton.swift

```diff
@@ -131,6 +158,34 @@ struct NoteButton: View {
                 buttonOrLabel
             }
         }
+        .onAppear {
+            if fetchReplies {
+                subscribeToReplies()
+            }
+        }
+        .onDisappear {
+            relaySubscriptions.removeAll()
+        }
+    }
+
+    /// Open relays subscriptions asking one reply from anyone and up to four
+    /// replies from follows.
+    func subscribeToReplies() {
+        Task(priority: .userInitiated) {
+            // Close out stale requests
+            relaySubscriptions.removeAll()
+            relaySubscriptions.append(
+                await relayService.requestReplyFromAnyone(
+                    for: note.identifier
```

Comment:
🤔 this code appears to be working, but I'm pretty sure we want to use `displayedNote` here and in the other call to relayService. In the case of a repost `note` will be the kind 6 and `displayedNote` will be the kind 1 we want to fetch replies for.
```suggestion
                    for: displayedNote.identifier
```

Reply:
You are right. I missed that after moving the code to NoteButton.